fluent-package/apt/systemd-test/install-newly.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -exu
+
+. $(dirname $0)/../commonvar.sh
+
+sudo apt install -V -y \
+    /vagrant/${distribution}/pool/${code_name}/${channel}/*/*/*_${architecture}.deb
+
+systemctl status fluentd
+
+sudo apt remove -y fluent-package
+
+test $(systemctl status fluentd > /dev/null 2>&1; echo $?;) -eq 4
```

Comment:
It may be readable not to use magic number 4 (assign meaningfull variable and refer it to express your intention)

Reply:
or use systemctl is-enabled or systemctl is-active predicate appropriately. (it depends on context)

Reply:
Thanks! I see!

Now I noticed this test always fails because the service is uninstalled by `apt remove`, not `apt purge`.

So I simplified this.
I felt there was little need to check the `unit not found` result after being purged.